app/uk/gov/hmrc/teamsandrepositories/connectors/JenkinsConnector.scala

```diff
@@ -134,70 +124,28 @@ class JenkinsConnector @Inject()(
     httpClientV2
       .post(url)
       .setHeader("Authorization" -> authorizationHeader)
-      .execute[JenkinsBuildData]
+      .execute[BuildData](readFromJson(BuildData.jenkinsReads, implicitly[Manifest[BuildData]]), ec)
```

Comment:
same here. If it's needed in multiple functions, it could be a private implicit in `JenkinsConnector`